gslib/context_config.py

```diff
@@ -263,5 +263,4 @@ def get_context_config():
   Returns:
     ContextConfig or None if global singleton doesn't exist.
   """
-  global _singleton_config
```

Comment:
Is this b/c of Python 2 deprecation? But how did it work in Python 3 before if this is causing bugs?

Reply:
I am not sure why this started popping up now, but our pylint started complaining about this. Ideally this should be a separate PR, but I didn't want to block this PR for a 3 line change given the urgency of this PR, so made the change here.